CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### New Features
 
+ * [#841](https://github.com/toptal/chewy/pull/841): Add the [`collapse`](https://www.elastic.co/guide/en/elasticsearch/reference/current/collapse-search-results.html) option to the request. ([@jkostolansky][])
```

Comment:
Could you please also add a link to your GH profile at the bottom of this file, in alphabetical order, I believe this shall be line#699?

Reply:
I'm sorry, I forgot to add that line. It's fixed now.